<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QDialog" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>211</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_Test">
      <property name="text">
       <string>Тест</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_SpeedLine">
      <property name="text">
       <string>Усоритель чтения</string>
      </property>
      <property name="autoRepeatInterval">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_Ex">
      <property name="text">
       <string>Упражения</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_Exit">
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Меню</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_username">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_wm_rating">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_score">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
